test/unit/test-multipolicy.cpp

```diff
@@ -22,44 +22,48 @@
 
 // Tag type to dispatch to test bodies based on policy selected by multipolicy
 
-struct mp_test_body;
+//struct mp_test_body;
+
 namespace test_policy
 {
 template <int i>
 struct mp_tag {
 };
-// fake forall_impl overload to test multipolicy dispatch
-template <int i, typename Iterable>
-void forall_impl(const mp_tag<i> &p, Iterable &&iter, mp_test_body const &body)
-{
-  body(p, iter.size());
-}
 }
 
-using test_policy::mp_tag;
```

Comment:
@rhornung67 I think this was supposed to be structured like this to ensure the correct mp_tag was picked up below, rather than explicitly using the entire namespace. @trws ?

Reply:
Compilers were complaining about mp_test_body being an incomplete type. I don't think this change affects what the code was supposed to do. I can change it back if @trws thinks it should be the way it was. Then, we will always see a compiler warning.

Reply:
@trws are you OK with this change?

Reply:
This line is still there, it's just on line 63 now isn't it?  This specific change is just cosmetic, so it really isn't a big issue if you prefer this @rhornung67, but if it was causing warnings why isn't the same thing causing warnings [below](https://github.com/LLNL/RAJA/pull/350/files/1b017ffbe2dee31ce155ac5cf70416774b84f1e8#diff-b6ce138c68f23744b3fcbc496baa0b30R66)?

Reply:
@trws I don't have a preference. This is what I came up with to squash the compiler warning and all tests passed so I went with it.